docs/UserGuide.md

```diff
@@ -2,196 +2,135 @@
 layout: page
 title: User Guide
 ---
-
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
-
+lesSOn is a **flashcard software aimed to help individuals with their memory work in school, optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, lesSon can get your contact management tasks done faster than traditional GUI apps.
```

Comment:
Can we standardise the spelling of lesSON?

Reply:
Sure! my bad overlooking this